maintainers/maintainer-list.nix

```diff
@@ -5181,6 +5181,12 @@
     githubId = 13689192;
     name = "Nguyễn Gia Phong";
   };
+  mcwitt = {
+    email = "mcwitt@gmail.com";
+    github = "mcwitt";
+    githubId = 319411;
+    name = "Matt Wittmann";
+  };
```

Comment:
Please make this into a separate commit:

```
maintainers: add mcwitt
```

Reply:
Done in d988033.